data/wanderer ships.txt

```diff
@@ -782,5 +782,5 @@ ship "Riptide"
 	explode "medium explosion" 40
 	explode "large explosion" 20
 	"final explode" "final explosion medium"
-	description "Because of the Unfettered's increasing aggression the Riptide was designed to quickly move the remaining Wanderer population beyond The Eye."
+	description "Because of the Unfettered's increasing aggression "The Riptide" was designed to quickly move the remaining Wanderer population beyond The Eye."
```

Comment:
He said put a comma before it: `aggression, the Riptide`.
1) no ship in the game puts its own name in quotes.
2) Even if they did, you'd need to replace the quotes around the entire description into backticks